src/Card/CardStatus.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Icon } from '..';
+
+const CardStatus = React.forwardRef(({
+  className,
+  children,
+  variant,
+  icon,
+  title,
+}, ref) => (
+  <div
+    className={classNames('pgn__card-status',
+      `alert-${variant}`, 'alert', 'alert-content',
```

Comment:
I'm still a bit hesitant to rely on the `alert`-related class names, even though the styles are similar/identical as it creates a coupling between `Card.Status` and `Alert` from a styles perspective that we might not actually want. For example, if we change `Alert` styling, it'll also impact `Card.Status`.

I think I'd still be in favor of not using any `alert-*` related class names from Bootstrap, in favor of replicating those styles specifically with `pgn` prefixed class names.

Reply:
Now all `alert-*` are removed and it is `pgn__card-status__{variant}`, `pgn__card-status__heading` etc. All `alert` styles are transferred to `pgn__card-status`